packages/volto/news/7023.documentation

```diff
@@ -0,0 +1 @@
+Corrected Redux-related docs. Improved word wrapping to column 80 in source code of Redux-related docs and added link to Redux development addon for Firefox. @silviubogan
```

Comment:
Again, please remove the line wrapping at 80 characters, and use one sentence per line instead. See https://6.docs.plone.org/contributing/documentation/authors.html#plone-documentation-styleguide for an explanation. Thank you!

```suggestion
Corrected Redux-related docs. Removed word wrapping in Redux-related docs, and added link to Redux development add-on for Firefox. @silviubogan
```